src/components/tree/InfoNode.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC } from 'react'
+
+interface Props {
+  node: TreeInfoNode,
+}
+
+const InfoNode: FC<Props> = ({ node }) => {
```

Comment:
Also the concept of InfoNode is a bit off. It would be better to have specific nodes like EmptyNode or LoadMoreNode so we can apply a specific styling.